doc/nrf/protocols/thread/sed_ssed.rst

```diff
@@ -0,0 +1,63 @@
+.. _thread_sed_ssed:
+
+Sleepy End Device types in Thread
+#################################
+
+Sleepy End Devices (SEDs) and Synchronized Sleepy End Devices (SSEDs) are both :ref:`Minimal Thread Devices (MTDs)<thread_types_mtd>`.
+For more general information about Thread device types, see the documentation for :ref:`thread_device_types`.
```

Comment:
```suggestion
Sleepy End Devices (SEDs) and Synchronized Sleepy End Devices (SSEDs) are both Minimal Thread Devices (MTDs).
Unlike Full Thread Devices (FTDs), MTDs do not maintain a routing table and are typically low-power devices that are not always on.

For more general information about Thread device types, see the documentation for :ref:`thread_device_types`.
```
I would include some extra info and remove one of the links, so that we don't push the reader to go to other page if he needs a refresher on the differences between MTDs and FTDs.

Reply:
Moved the links around a little.